pkg/controller/configmap/config.go

```diff
@@ -256,7 +249,30 @@ func Render(configmap *corev1.ConfigMap, ncpReplicas int32, nsxSecret *corev1.Se
 	return objs, nil
 }
 
-func HasNetworkConfigChange(currConfig *configv1.Network, prevConfig *configv1.Network) bool {
+func (r *ReconcileConfigMap) GetManifestDir() (string, error) {
+	release, err := os.Open("/host/etc/os-release")
+	if err != nil {
+		return "", err
+	}
+	defer release.Close()
+	content, err := ioutil.ReadAll(release)
+	osRelease := string(content)
+	if strings.Contains(osRelease, "CoreOS") {
+		// TODO: add platform detection logic if we allow the user deploy coreos image on native k8s
```

Comment:
I think QE has test to run CoreOS on K8s, but we should still use rhel_on_kubernetes manifests for it, right?

Reply:
I checked the NCP bootstrap code, the CoreOS related logic is added by Michal and only for openshift support. I double confirmed that with QE. So we don't need to care about CoreOS on K8s.